Assets/Scripts/Framework/GeoLocation/RouteHighlighter.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Framework.GeoLocation
+{
+    [RequireComponent(typeof(LineRenderer))]
+    public sealed class RouteHighlighter : MonoBehaviour
+    {
+        private const float INVOKE_DELAY = 0.1f;
+
+        [SerializeField] private UnityEvent withinRange = new UnityEvent();
+        
+        [SerializeField] private List<Transform> routePoints;
+        [SerializeField] private LineRenderer lineRenderer;
+        [SerializeField] private float closeRange = 5f;
+        
+        [SerializeField] private Transform _nextPoint;
```

Comment:
1. Events onderaan de varbiles - [convention](https://github.com/Team-Swamp/Codename-BDLMTW/wiki/Code-convention#classes)
2. `_nextPoint` renaming of true private van maken

```suggestion
        private const float INVOKE_DELAY = 0.1f;
        
        [SerializeField] private List<Transform> routePoints;
        [SerializeField] private LineRenderer lineRenderer;
        [SerializeField] private float closeRange = 5f;
        
        [SerializeField] private Transform nextPoint;
        // or
        private Transform _nextPoint;
        
        [SerializeField] private UnityEvent withinRange = new UnityEvent();
```